packages/spor-badge-react-native/src/Badge.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Text } from "@vygruppen/spor-typography-react-native";
+import { Box } from "@vygruppen/spor-layout-react-native";
+import { Theme } from "@vygruppen/spor-theme-react-native";
+import {
+  SpacingProps,
+  SpacingShorthandProps,
+  VariantProps,
+  createVariant,
+  useRestyle,
+  composeRestyleFunctions,
+  spacing,
+  spacingShorthand,
+} from "@shopify/restyle";
+
+type Variant = VariantProps<Theme, "badgeColorSchemes", "variant">;
+const variant = createVariant({ themeKey: "badgeColorSchemes" });
+
+type RestyleProps = SpacingProps<Theme> &
+  SpacingShorthandProps<Theme> &
+  Variant;
+
+const restyleFunctions = composeRestyleFunctions<Theme, RestyleProps>([
+  spacing,
+  spacingShorthand,
+  variant,
+]);
+
+type ColorVariants =
+  | "yellow"
+  | "light-yellow"
+  | "red"
+  | "green"
+  | "orange"
+  | "blue"
+  | "grey"
+  | "white";
+
+type BadgeProps = Exclude<RestyleProps, "variant"> & {
+  children?: string;
+  variant: ColorVariants;
+  borderOutline: boolean;
```

Comment:
React-implementasjonen av Badge kaller dette for `colorScheme` istedenfor `variant`, og det tenker jeg er vel så greit å gjøre her.

React-implementasjonen har også løst `borderOutline` med å innføre `variant` av enten "solid" eller "outline". Det kunne man jo fint gjort her også.

Reply:
Ja enig! Vi tenkte det samme men slet med implementasjonen fordi det virket som VariantProps kun ville godta "variant", men fant nå at det kun var fordi vi ikke hadde definert property i createVariant. Fikset opp i nå :)